osquery/core/conversions.cpp

```diff
@@ -48,11 +50,14 @@ std::string base64Decode(const std::string& encoded) {
   if (size == 0) {
     return "";
   }
-
-  std::copy(base64_dec(is.data()),
-            base64_dec(is.data() + size),
-            std::ostream_iterator<char>(os));
-
+  try {
+    std::copy(base64_dec(is.data()),
+              base64_dec(is.data() + size),
+              std::ostream_iterator<char>(os));
+  } catch (const boost::archive::iterators::dataflow_exception& e) {
+    LOG(INFO) << "Could not base64 decode string: " << e.what();
```

Comment:
This could be caused by a lot of things, like using the unbase64 on the wrong column and you'll get one log line per row, I felt like that would cause too many warnings and dropped it to an info